draft-ietf-core-groupcomm-bis.md

```diff
@@ -758,7 +764,7 @@ Note that a client sending an IPv4 multicast CoAP message to a port number that
 
 ### TCP, TLS, and WebSockets ###
 
-Because it supports unicast only, {{RFC8323}} (CoAP over TCP, TLS, and WebSockets) has a restricted scope as a transport for CoAP group communication. This is limited to the use of block-wise transfer discussed in {{sec-block-wise}}.
+Because it supports unicast only, employing {{RFC8323}} (CoAP over TCP, TLS, and WebSockets) as a transport for CoAP group communication is limited to the use of block-wise transfer discussed in {{sec-block-wise}}.
```

Comment:
Thank you for improving the wording.
Couldn't CO transports still be used as the first hop to a proxy that then uses IP multicast?

Reply:
Yes. What about adding the following new paragraph at the end of the same Section 3.9.4 "TCP, TLS, and WebSockets" ?

> Alternatively, a suitable cross-proxy can be set up, such that it receives a unicast CoAP group request over TCP/TLS/WebSockets, and then forwards the request to the servers in the group over IP multicast (see {{sec-proxy}}). The discovery of such a proxy can also rely on means defined in {{I-D.ietf-core-transport-indication}}.

Reply:
I'm not sure there is an "alternative" here.  I would just say that 8323 is useful in unicast situations, such as

* completing block-wise
* talking to a proxy

Reply:
Thanks!

Now in https://github.com/core-wg/groupcomm-bis/pull/44/commits/2327d509f69153ed9c23fcc8aaf3542bae2fadfa  , where the main point is stated upfront. @cabo @EskoDijk , please have a look.

Reply:
The RFC editor will provide some minor wording improvements, so I think we can ship this now.

Reply:
@marco-tiloca-sics I'm also okay with the new text - better now.